2.2.5/bullseye/build-time-pip-constraints.txt

```
@@ -0,0 +1,558 @@
+APScheduler==3.6.3
+Authlib==1.0.0
+Babel==2.9.1
+Deprecated==1.2.13
+Flask-AppBuilder==3.4.4
+Flask-Babel==2.0.0
+Flask-Bcrypt==0.7.1
+Flask-Caching==1.10.1
+Flask-JWT-Extended==3.25.1
+Flask-Login==0.4.1
+Flask-OpenID==1.3.0
+Flask-SQLAlchemy==2.5.1
+Flask-Session==0.4.0
+Flask-WTF==0.14.3
+Flask==1.1.2
+GitPython==3.1.27
+HeapDict==1.0.1
+JPype1==1.3.0
+JayDeBeApi==1.2.3
+Jinja2==3.0.3
+Mako==1.2.0
+Markdown==3.3.6
+MarkupSafe==2.0.1
+PyGithub==1.54.1
+PyHive==0.6.5
+PyJWT==1.7.1
+PyNaCl==1.5.0
+PyYAML==5.4.1
+Pygments==2.11.2
+Rx==3.2.0
+SQLAlchemy-JSONField==1.0.0
+SQLAlchemy-Utils==0.38.2
+SQLAlchemy==1.3.24
+SecretStorage==3.3.1
+Sphinx==4.5.0
+Unidecode==1.3.4
+WTForms==2.3.3
+Werkzeug==1.0.1
+adal==1.2.7
+aiohttp==3.8.1
+aiosignal==1.2.0
+alabaster==0.7.12
+alembic==1.7.7
+aliyun-python-sdk-core==2.13.36
+aliyun-python-sdk-kms==2.15.0
+amqp==5.1.0
+analytics-python==1.4.0
+ansiwrap==0.8.4
+anyio==3.5.0
+apache-airflow-providers-airbyte==2.1.4
```

```suggestion
apache-airflow-providers-airbyte==1!2.1.4
```

These need the epoch.